test/texture2d.js

```diff
@@ -974,16 +975,17 @@ tape('texture 2d', function (t) {
         width: 5,
         height: 5,
         pixels: [
-          255, 0, 0, 255, 255, 255, 255, 255, 0, 0, 0, 255, 0, 0, 0, 255, 0, 0, 0, 0,
-          255, 255, 255, 255, 255, 255, 255, 255, 0, 0, 0, 255, 0, 0, 0, 255, 0, 0, 0, 0,
-          0, 0, 0, 255, 0, 0, 0, 255, 0, 255, 0, 255, 0, 255, 0, 255, 0, 0, 0, 0,
-          0, 0, 0, 255, 0, 0, 0, 255, 0, 255, 0, 255, 0, 255, 0, 255, 0, 0, 0, 0,
-          0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0
+          255, 0, 0, 255, 255, 255, 255, 255, 0, 0, 0, 255, 0, 0, 0, 255, null, null, null, null,
```

Comment:
`null` signifies insignificance

Reply:
[Noted in the comment above](https://github.com/regl-project/regl/pull/479/files/070376583bf0206474e1928a036b0d62b89f730a..16b7628c24d02b83ffc73d7bda4ccd518f4ad2f8#diff-9e30d22e73f1080624632e66b0a55e32R165), so sounds good 👍